<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1041</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ejercicio 1</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nota:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarAlArchivo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar al archivo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="leerElArchivo">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Leer el archivo</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>130</y>
        <width>351</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimirButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarAlArchivoSiPasaron">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar al archivo si pasaron</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ejercicio 2</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Precio:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>372</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cantidad:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimirButton_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>230</y>
        <width>351</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarAlArchivo_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar al archivo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="leerElArchivo_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Leer el archivo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Codigo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>360</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nueva Cantidad:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modificarButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ejercicio 3</string>
     </attribute>
     <widget class="QPushButton" name="leerArhivoCSV">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Leer Archivo CSV</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>150</y>
        <width>351</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimirButton_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="guardarArhivoCSV_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar en archivo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="agregar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cliente:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Codigo:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
